changelog/@unreleased/pr-132.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: Accept-funcs flag correctly passed through to conjure-go generator
```

Comment:
Can you expand on this a bit more? In the case of the changelog, this content is going to be part of the release notes, so it should be written from the perspective that would provide the most context for that.

For example, is this something like:

```
Fixes issue where the "accept-func" configuration was not respected.

Previously, the "accept-funcs" flag was not properly passed through to the conjure-go generator, which made it such that specifying the "accept-funcs" field in configuration had no effect.
```

It doesn't have to be this exact wording, but basically it's most helpful if it's possible to tell the actual user-visible change from reading the release notes (whether to include the implementation details of the fix as part of the notes or just in the commit message is more of a subjective thing and could go either way).